lib/app_profiler.rb

```diff
@@ -57,5 +58,9 @@ def request_profile_header
     def profile_data_header
       @@profile_data_header ||= profile_header.dup << "-Data" # rubocop:disable Style/ClassVars
     end
+
+    def profile_url_formatter=(prok)
+      @@profile_url_formatter = prok # rubocop:disable Style/ClassVars
```

Comment:
Pretty sure class variable is not what you want. Use `@profile_url_formatter` instead.

The above can be cleaned up as well.

Reply:
There's quite a few on this class, should I fix all of them in this PR?

Reply:
Up to you.